includes/SiteCapabilities.php

```diff
@@ -57,7 +57,7 @@ public function all(): array {
 			$this->transient->set( 'nfd_site_capabilities', $capabilities, 4 * constant( 'HOUR_IN_SECONDS' ) );
 		}
 
-		return $capabilities;
+		return $capabilities ? $capabilities : array();
```

Comment:
@circlecube Looking at the code, it seems the fetch method should always return an array. Is the transient get returning null instead of false somehow? I think this fix would work, I just want to make sure we understand and address any root cause.

Reply:
Right, I've tested in a site on 2 other hosts (without migrating though) and can't reproduce this error. I'm finding valid capabilities array at the endpoint and stored in the transient. The only way I expect a `null` value can be returned is if it came from the transient, which I wouldn't expect is possible. Maybe a filter?

Reply:
Was able to reproduce the error by setting the `nfd_site_capabilities` transient to `null`. Not sure how it would be set unless it was part of some cloning/migration script. This should address the edge-case though.

Reply:
@circlecube - FYI, this PR was opened: https://github.com/newfold-labs/wp-module-data/pull/123